packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -485,121 +458,78 @@ describe('Checkout', () => {
-                        ...defaultCheckoutState,
-                        isLoggedIn: true
-                    }),
-                    i18n,
-                    localVue,
-                    propsData,
-                    data () {
-                        return {
-                            nonFulfillableError: {
-                                code: ERROR_CODE_FULFILMENT_TIME_UNAVAILABLE,
-                                shouldShowInDialog: false
-                            }
-                        };
-                    }
-                });
-
-                // Act
-                const result = wrapper.vm.shouldShowErrorDialog;
-
-                // Assert
-                expect(result).toBe(false);
-            });
+                const dialog = {
+                    name: 'error-dialog'
+                };
 
-            it('should return `false` if there is no error', () => {
-                // Arrange
                 const wrapper = shallowMount(VueCheckout, {
                     store: createStore({
                         ...defaultCheckoutState,
-                        isLoggedIn: true
+                        message
                     }),
                     i18n,
                     localVue,
                     propsData
                 });
 
-                // Act
-                const result = wrapper.vm.shouldShowErrorDialog;
-
-                // Assert
-                expect(result).toBe(false);
+                it('should return dialog', () => {
+                    // Assert
+                    expect(wrapper.vm.messageType).toEqual(dialog);
+                });
             });
-        });
 
-        describe('restaurantMenuPageUrl ::', () => {
-            it('should return the URL to redirect back to the restaurant menu', () => {
+            describe('when a message exists AND `shouldShowInDialog` is false', () => {
                 // Arrange
-                const restaurantSeoName = 'masala-zone-camden';
+                const alertMessage = 'Something went wrong, please try again later';
+
+                const alert = {
+                    name: 'alert',
+                    props: {
+                        type: 'danger',
+                        class: 'c-checkout-alert',
+                        heading: 'Error'
+                    },
+                    content: alertMessage
+                };
+
                 const wrapper = shallowMount(VueCheckout, {
                     store: createStore({
                         ...defaultCheckoutState,
-                        isLoggedIn: true,
-                        restaurant: {
-                            seoName: restaurantSeoName
-                        }
+                        message: alertMessage
                     }),
                     i18n,
                     localVue,
-                    propsData
+                    propsData,
+                    mocks: {
+                        $style
```

Comment:
Do we need this?

Reply:
We do. I have a test that when the `alertMessage` is passed a prop with a class in. Without the `$style` mock it comes back as undefined